webrtc/README.md

```diff
@@ -0,0 +1,12 @@
+# WebRTC
+
+This directory contains the WebRTC test suite.
+
+## Attribution
```

Comment:
This looks great! To avoid the possible impression that this is fulfilling a license requirement for attribution (such as with CC-by) may I suggest "Acknowledgments" instead?